moveit_core/CMakeLists.txt

```diff
@@ -50,6 +50,14 @@ find_package(ruckig REQUIRED)
 # in include_directories below because the target is correctly imported here.
 get_target_property(ruckig_INCLUDE_DIRS ruckig::ruckig INTERFACE_INCLUDE_DIRECTORIES)
 set(ruckig_LIBRARIES "ruckig::ruckig")
+# ruckig built with cloud client (default) does not automatically pull in nlohmann_json
+# and MoveIt can fail to build in install workspaces because of this.
+# Remove the WITH_CLOUD_CLIENT definition because we don't use it anyway.
```

Comment:
The explanation here is not clear to me. I suggest limiting it to the minimum here and refer to the PR for more details. Ideally, refer to the upstream PR instead.
```suggestion
# Remove WITH_CLOUD_CLIENT from ruckig's compile definitions to avoid build issues
# See  https://github.com/moveit/moveit/pull/3636 for details
```